### DISCUSSION TITLE
Dragging items on mobile

### DISCUSSION THREAD

**Discussion by giuseppetomassetti** [ivan-lednev/obsidian-day-planner#474] [Q&A]
On the desktop app I can draw events on the calendar. On the mobile app I can only create new events by tapping on the screen, but pressing and holding an event to drag it doesn't seem to work. I also tried double tapping, using two fingers etc, to no avail...

Am I missing something?

Thank you!

Giuseppe

**Comment by ivan-lednev** [comment]
This is not yet implemented. The progress can be tracked here: #340
